EIPS/eip-7377.md

```diff
@@ -0,0 +1,127 @@
+---
+eip: 7377
+title: Migration Transaction
+description: Allow EOAs to send a one-time transaction which deploys code at their account.
+author: lightclient (@lightclient), Sam Wilson (@samwilsn), Ansgar Dietrichs (@adietrichs)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-migration-transaction/15144
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-07-21
+requires: 170, 1559, 2718
```

Comment:
```suggestion
requires: 170, 1559, 2718, 3607
```
You mentioned EIP-3607 should be used to avoid multiple migration transactions (which would incidentally not work as intended as all the memory is incremented). EIP-3607 should be added to the requires list.

Reply:
It isn't required to implement the EIP though. I just wanted to bring it to the reader's attention that this will be the behavior given 3607 exists on mainnet and there are some useful properties of it.

Reply:
Yes, but this EIP depends on that specific EIP-3607 behavior. I would argue that means this EIP requires the other EIP, since otherwise there are unintended consequences.

Reply:
EIP-1 states: 

> "A requires dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Merely mentioning another EIP does not necessarily create such a dependency."

This EIP can both be understood and implemented without the concept / technical element from EIP-3607, therefore it is not a requirement.